MapboxCoreNavigation/RouteController.swift

```diff
@@ -196,7 +207,6 @@ open class RouteController: NSObject {
     deinit {
         suspendLocationUpdates()
         checkAndSendOutstandingFeedbackEvents(forceAll: true)
-        sendCancelEvent()
```

Comment:
Should this still happen?

Reply:
Gah -- that was an oversight.

Reply:
Correction: it kinda was an oversight. a "Navigation Cancel Event" is sent by the EORVC when you hit "End Navigation", but we still need to send a "blank" cancel event if the user sets `showsEndOfRoute` to `false`. Updating PR now.

Reply:
What occurs if the user is only using `MapboxCoreNavigation`? There will be no `EORVC `.